### PR TITLE
Make it possible to get current traffic config

### PR DESCRIPTION
**Pull request by mikkeloscar**
This adds the options to run `zdeploy traffic $APPLICATION` without the `$RELEASE` and `$PERCENT` parameters in order to just read the current traffic weights for the ingress named `$APPLICATION`.

The output will be json: `{"mysvc-1": 30, "mysvc-2": 70}`.

### CODE REVIEW

File: zalando_deploy_cli/cli.py [hunk 1]
```diff
@@ -537,24 +565,51 @@ def traffic(config, application, release, percent, execute):
...
+            if path['backend']['serviceName'] in backend_weights:
+                weight = backend_weights[path['backend']['serviceName']]
+                sc['sum'] += weight
+            else:
+                sc['count'] += 1
+
+        # calculate traffic weight for each backend
+        for path in rule['http']['paths']:
+            if path.get('path', '') in path_sum_count:
+                p = path.get('path', '')
```
Comment (hjacobs) [inline zalando_deploy_cli/cli.py]: you can move this line before the if and use `p` in the if condition..

File: zalando_deploy_cli/cli.py [hunk 2]
```diff
@@ -537,24 +565,51 @@ def traffic(config, application, release, percent, execute):
...
+                sc = path_sum_count[p]
+                backend = path['backend']['serviceName']
+                if backend in backend_weights:
+                    weight = backend_weights[backend]
+                    backends[backend] = int(weight / sc['sum'] * 100)
```
Comment (hjacobs) [inline zalando_deploy_cli/cli.py]: might be division by zero

File: zalando_deploy_cli/cli.py [hunk 3]
```diff
@@ -537,24 +565,51 @@ def traffic(config, application, release, percent, execute):
...
+
+    backends = {}
+
+    for rule in ingress['spec']['rules']:
+        path_sum_count = {}
+
+        # get backend weight sum and count of all backends for all paths
+        for path in rule['http']['paths']:
+            if path.get('path', '') not in path_sum_count:
+                p = path.get('path', '')
```
Comment (hjacobs) [inline zalando_deploy_cli/cli.py]: same here move `p` before if

Comment (mikkeloscar) [inline zalando_deploy_cli/cli.py]: done.

File: zalando_deploy_cli/cli.py (hunk 2 above)
Comment (mikkeloscar) [inline zalando_deploy_cli/cli.py]: fixed

File: zalando_deploy_cli/cli.py (hunk 1 above)
Comment (mikkeloscar) [inline zalando_deploy_cli/cli.py]: done.

**Comment by hjacobs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**